bentoml/_internal/bento/build_config.py

```diff
@@ -515,15 +761,49 @@ def _converter(value: t.Union[OptionsCls, t.Dict[str, t.Any]]) -> options_type:
 
 @attr.define(frozen=True, on_setattr=None)
 class BentoBuildConfig:
-    """This class is intended for modeling the bentofile.yaml file where user will
-    provide all the options for building a Bento. All optional build options should be
-    default to None so it knows which fields are NOT SET by the user provided config,
-    which makes it possible to omitted unset fields when writing a BentoBuildOptions to
+    """
+    This class is intended for modeling the :code:`bentofile.yaml` file where user will
+    provide all the options for building a Bento.
+
+    All optional build options should be default to :code:`None` so BentoML knows
+    which fields are **NOT SET** by the user provided config.
+
+    Thus, it is possible to omitted unset fields when writing a :code:`BentoBuildConfig` to
     a yaml file for future use. This also applies to nested options such as the
-    DockerOptions class and the PythonOptions class.
+    :class:`DockerOptions` class and the :class:`PythonOptions` class.
+
+    Args:
+
+        service (:code:`str`):
+            The required name of the service to be built.
+
+        description (:code:`str`, `optional`):
+            An optional description of the service.
+
+        labels (:code:`dict[str, Any]`, `optional`):
+            An optional dictionary of labels to be applied to the service.
+
+        include (:code:`list[str]`, `optional`):
+            An optional list of files to include in the build. This is useful for only including necessary files for the bento.
+
+            By default if :code:`include` is not set under :code:`bentofile.yaml`, all files will be included.
+
+        exclude (:code:`list[str]`, `optional`):
+            An optional list of files to exclude from the build. This is useful for excluding unnecessary files from the bento.
+
+            By default if :code:`exclude` is not set under :code:`bentofile.yaml`, no files will be excluded.
+
+        docker (:class:`DockerOptions`, `optional`):
+            All options related to :class:`DockerOptions`.
+
+        python (:class:`PythonOptions`, `optional`):
+            All options related to :class:`PythonOptions`.
+
+        conda (:class:`CondaOptions`, `optional`):
+            All options related to :class:`CondaOptions`.
```

Comment:
this feels repetitive to https://docs.bentoml.org/en/latest/concepts/bento.html#advanced-project-structure
and the type info is pretty self-explanatory. let's remove it from here

```suggestion
```